parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/AbstractParallelEoSStreamProcessor.java

```diff
@@ -206,6 +206,9 @@ public static <K, V> ControllerEventMessage<K, V> of(WorkContainer<K, V> work) {
      */
     private Instant lastCommitTime;
 
+    private final RetryHandler<K, V> retryHandler;
```

Comment:
Do we need this reference here if it's stored in PCModule?

Reply:
Thanks, fixed